oga/primitives.py

```diff
@@ -69,5 +71,10 @@ def to_json(self) -> dict:
             "favorites": self.favorites,
             "files": [file.to_json() for file in self.files]}
 
+        if self.attribution != None:
+            ret["attribution"] = self.attribution
```

Comment:
same comment as above, I'd prefer `"attribution"` be included even when `None`.  stripping empty values at the CLI could be an option but the model shouldn't omit them.

Reply:
I don't mind either way is fine by me.